test_app/resource_api.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group
+
+from ansible_base.authentication.models import Authenticator
+from ansible_base.resource_registry.registry import ResourceConfig, ServiceAPIConfig, SharedResource
+from ansible_base.resource_registry.shared_types import TeamType, UserType
+
+
+class APIConfig(ServiceAPIConfig):
+    service_type = "aap"
+
+
+RESOURCE_LIST = (
+    ResourceConfig(get_user_model(), shared_resource=SharedResource(serializer=UserType, is_provider=False), name_field="username"),
+    ResourceConfig(
+        Group,
+        shared_resource=SharedResource(serializer=TeamType, is_provider=True),
+    ),
+    ResourceConfig(Authenticator),
```

Comment:
authenticators are not shared, right? Each component can have its own authenticators which may be different from all the others, and they might not even have the same plugins on the file system available, intentionally. But this system may, hypothetically, allow for sharing metadata for `aap.authenticators` and `awx.authenticators`.

Reply:
They aren't. I was just looking for existing models that I could hook in here for testing.